Tools/BTagCorrector.h

```diff
@@ -135,26 +135,13 @@ class BTagCorrectorTemplate
 	readerFastDown = BTagCalibrationReader(BTagEntry::OP_MEDIUM, "down");
 	readerFastDown.load(calibFast, BTagEntry::FLAV_B, "fastsim"); readerFastDown.load(calibFast, BTagEntry::FLAV_C, "fastsim");  readerFastDown.load(calibFast, BTagEntry::FLAV_UDSG, "fastsim"); 	
     }
-    void SetTreeNames(const NTupleReader& tr)
+    void SetVarNames(std::string MCBranchName, std::string JetsVecName, std::string BJetsVecName, std::string JetsFlavorName, std::string myVarSuffix = "")
     {
-        //Switch based on which nTuples are used
```

Comment:
These are Chris's changes, can you rebase to the latest master to make this diff understandable?